CONTRIBUTING.md

```diff
@@ -61,11 +61,11 @@ If you observe following error running any of the test files that means you are
 
 - To fix errors always focus on passing all minimal tests first without running mainnet tests.
 - Spec tests often compare full expected vs actual states in JSON format.
-- A single logical error can cause many spec tests to fail. To focus on a single test at a time you can use vitest's option `--bail` to stop at the first failed test
+- A single logical error can cause many spec tests to fail. To focus on a single test at a time you can use vitest's option `--bail 1` to stop at the first failed test
 - To then run only that failed test you can run against a specific file as use vitest's filters to run only one case
 
 ```sh
-LODESTAR_PRESET=minimal yarn vitest --run --config vitest.config.spec.ts test/spec/phase0/sanity.test.ts
+LODESTAR_PRESET=minimal yarn vitest --run --config vitest.base.spec.config.ts test/spec/phase0/sanity.test.ts
```

Comment:
This change looks wrong, you can't execute the command like this because `vitest.base.spec.config.ts` is in root but `test/spec/phase0/sanity.test.ts` means the command has to be executed in `packages/beacon-node`, I think the previous config file ref is correct

Reply:
Fixed and clarified so it is clear in which folder it should be run.